### ISSUE TITLE
[FEATURE] Custom location for GoogleService-Info.plist

### ISSUE THREAD

**Issue by zdm** [dpa99c/cordova-plugin-firebasex#452]
Hi,
I need to store all files, related to the ios platform, in resources/ios subdirectory.
So, I added file copy instruction to config.xml
```
<resource-file src="resources/ios/GoogleService-Info.plist" />
```

But it conflicts with the default firebasex behavior. 
When I run `cordova build ios` it dies with error about conflicting resources.

Is it possible to respect user customized settings and apply defaults only if there is no user configuration defined?

Thank you.

**Comment by dpa99c** [comment]
Attempting to detect if `GoogleService-Info.plist` has been deployed via another source is non-trivial since the execution order of plugin hook scripts is non-deterministic.
I can see no reason why you should need to reference `GoogleService-Info.plist` directly from the `config.xml`: if you place it in the project root directory, this plugin will deploy it to the iOS platform project.

**Comment by zdm** [comment]
It is good practice, when file in the project are placed in directories with the logical sense.
GoogleService-Info.plist is related only to ios platform and should be placed near other ios resources.
Maybe you can add option where we can specify GoogleService-Info.plist location?
If this is not very hard.
Thank you,

**Comment by dpa99c** [comment]
OK, this can be added as a feature request but it is not high on my priority list however I will consider accepting a PR which implements it for both the Android and iOS platforms.

**Comment by zdm** [comment]
By the way, `google-services.json` works for me without conflicts when it is located in `resources/android/google-services.json`.

**Comment by zdm** [comment]
I created PR, that closes this issue. Could you please review it?

**Comment by dpa99c** [comment]
Resolved by merge of PR #465
